packages/next/build/index.ts

```diff
@@ -587,6 +596,11 @@ export default async function build(dir: string, conf = null): Promise<void> {
             defaultMap[route] = { page }
           })
         })
+
+        if (useStatic404) {
+          defaultMap['/_errors/404'] = { page: '/_error' }
```

Comment:
Why not just `/404` 🤔

Reply:
Putting it in `/_errors/404` allows us to know for sure we can use it in `@now/next` as a static 404 page and prevents us from needing to worry about colliding with a `pages/404.js` file for now

Reply:
Can't we handle that by having a marker in the routes manifest already?

Reply:
Guess this is mostly related to preserving behavior that's there already while this is experimental.